app/views/settings_dialog.py

```diff
@@ -487,6 +487,12 @@ def _do_sorting_tab(self) -> None:
         explanatory_label.setWordWrap(True)
         tab_layout.addWidget(explanatory_label)
 
+        # Checkboxes
+        self.db_use_dependency_for_sorting_checkbox = QCheckBox(
+            "Use Mods Dependency Rules For Sorting"
```

Comment:
A little unclear. I'd add some more context to ensure that it is clear.

Something like, "Use mod dependency rules for sorting such that dependencies are loaded before the dependent mod."